rc/filetype/elixir.kak

```diff
@@ -92,12 +92,25 @@ define-command -hidden elixir-trim-indent %{
     try %{ execute-keys -draft -itersel <a-x> s \h+$ <ret> d }
 }
 
-define-command -hidden elixir-insert-on-new-line %{
-    evaluate-commands -draft -itersel %{
-        # copy -- comments prefix and following white spaces
-        try %{ execute-keys -draft k <a-x> s ^\h*\K--\h* <ret> y gh j P }
-    }
-}
+define-command -hidden elixir-insert-on-new-line %[
+    evaluate-commands -no-hooks -draft -itersel %[
+        # copy '#' comment prefix and following white spaces
+        try %{ exec -draft k <a-x> s ^\h*#\h* <ret> y jgh P }
```

Comment:
This creates wrong indentation for indented comments (like #1118). Test case:
```
def test do
    # comment%()
end
```
`c<ret>` will add spurious indentation to the new line.

Going back to the old regex with `\K` fixes it.

Reply:
Thanks a lot for pointing this out. I fixed it, added tests to cover that case and did similar fixes to adjacent Ruby PR https://github.com/mawww/kakoune/pull/4217